docs/sources/setup-grafana/configure-security/configure-scim-provisioning/manage-users-teams/_index.md

```diff
@@ -0,0 +1,190 @@
+---
+description: Learn how to implement SCIM provisioning in Grafana for automated user and team synchronization. SCIM integrates with identity providers like Okta and Azure AD to streamline user management, automate team provisioning, and replace Team Sync.
+keywords:
+  - grafana
+  - scim
+  - provisioning
+  - user-management
+  - team-management
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Manage users and teams with SCIM
+title: Manage users and teams with SCIM
+weight: 310
+---
+
+# Manage users and teams with SCIM
+
+{{< admonition type="note" >}}
+Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+{{< /admonition >}}
+
+SCIM streamlines identity management in Grafana by automating user lifecycle and team membership operations. This guide explains how SCIM works with existing Grafana setups, handles user provisioning, and manages team synchronization.
```

Comment:
nit: I'd suggest for now remove any reference to team management until we're certain that it'll be available for release.

Reply:
I am planning to not merge the base PR until we have clear understanding of the available functionality. So if you don't mind, we can keep parts related to the teams, merge them in the base PR and if we see it is not getting to the release, we can revert from the base PR. This will help to avoid any additional work.

Reply:
That sounds good enough!